src/components/o-modal/_template.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+
+export default ({ open }, childrenFragment) => {
+  const closeModal = () => {
```

Comment:
I would put this logic into the `index.js` as this is just templating. But its just an opinion